lib/lotus/action/head.rb

```diff
@@ -26,6 +26,8 @@ def finish
 
         if _requires_no_body?
           @_body = nil
+        else
+          @headers.merge!(configuration.default_headers)
```

Comment:
I added this else for merging the default headers if the method isn't HEAD or the status code isn't HTTP_STATUSES_WITHOUT_BODY defined above